src/views/dashboard.handlebars

```diff
@@ -0,0 +1,241 @@
+<div class="container">
+
+	<!-- Nav Container -->
+	<nav class="level">
+
+	  <!-- Left Side -->
+	  <div class="level-left">
+	    <div class="level-item">
+	      <p class="subtitle is-3">
+	        <strong>Devdash</strong>
+	      </p>
+	    </div>
+	  </div>
+
+	  <!-- Right side -->
+	  <div class="level-right">
+
+	  	<!-- Username -->
+	  	<p class="level-item">
+	  		<div class="field" id="form_id">
+  	  			<p class="control has-icons-left has-icons-right">
+      				<input class="input" id="email" type="email" placeholder="Email">
+      					<span class="icon is-small is-left">
+      						<i class="fas fa-envelope"></i>
+      					</span>
+      					<span class="icon is-small is-right">
+      						<i class="fas fa-check"></i>
+      					</span>
+      			</p>
+      		</div>
+      </p>
+	    
+	    <!-- Password -->
+	    <p class="level-item">
+	    	<div class="field" id="form_id">
+  				<p class="control has-icons-left">
+    				<input class="input" id="password" type="password" placeholder="Password">
+    					<span class="icon is-small is-left">
+      						<i class="fas fa-lock"></i>
+    					</span>
+  				</p>
+			</div>
+		</p>
+	    
+	    <!-- Login Button -->
+	    <p class="level-item">
+	    	<div class="field">
+  				<p class="control">
+    				<button class="button is-success" onclick="submitLogin()">
```

Comment:
The login route is `/user/auth/github`, and shouldn't need a function. This could be turned into a link styled as a button.